decidim-core/lib/decidim/exporters/pdf.rb

```diff
@@ -15,42 +15,39 @@ class PDF < Exporter
       #
       # Returns an ExportData instance.
       def export
-        html = controller.render_to_string(
-          template:,
-          layout:,
-          locals:
-        )
+        composer.styles(**styles)
 
-        document = WickedPdf.new.pdf_from_string(html, orientation:)
+        add_data!
 
-        ExportData.new(document, "pdf")
+        ExportData.new(composer.write_to_string, "pdf")
       end
 
-      # may be overwritten if needed
-      def orientation
-        "Portrait"
-      end
+      protected
 
-      # implementing classes should return a valid ERB path here
-      def template
-        raise NotImplementedError
+      delegate :document, to: :composer
+      delegate :layout, to: :document
+
+      def font
+        @font = load_font("source-sans-pro-v21-cyrillic_cyrillic-ext_greek_greek-ext_latin_latin-ext_vietnamese-regular.ttf")
```

Comment:
You're instantiating this variable but not accessing it. Maybe your intention was to memoize it? 
```suggestion
        @font ||= load_font("source-sans-pro-v21-cyrillic_cyrillic-ext_greek_greek-ext_latin_latin-ext_vietnamese-regular.ttf")
```

Reply:
Yep, memoization was in my head